hooks/shapediver/viewer/interaction/useHoverManager.ts

```diff
@@ -0,0 +1,56 @@
+import { IInteractionParameterSettings, MaterialStandardData } from "@shapediver/viewer";
+import { HoverManager } from "@shapediver/viewer.features.interaction";
+import { useRef, useEffect } from "react";
+import { useShapeDiverStoreViewer } from "shared/store/useShapeDiverStoreViewer";
+import { useInteractionEngine } from "./useInteractionEngine";
+
+// #region Functions (1)
+
+/**
+ * Hook allowing to create the hoverManager.
+ * 
+ * @param viewportId 
+ */
+export function useHoverManager(viewportId: string, settings?: IInteractionParameterSettings): {
```

Comment:
@MajorMeerkatThe3rd let's not use IInteractionParameterSettings here, but the type of settings that make sense for the HoverManager. Same for other hooks.